app/js/components/manageApps/SingleAddon.jsx

```diff
@@ -52,13 +78,23 @@ export default class SingleAddon extends React.Component{
         <td>
           <div className="addon-name">
             {
-              !app.install ?
-                app.appDetails.started === true || app.appDetails.started === undefined ?
-                  <div className="status-icon" id="started-status" />
-                  :
-                  <div className="status-icon" id="stopped-status" />
-                : null
+              app.appType === "owa"?
+                <div className="status-icon" id="started-status"/>
+                :!app.install ?
```

Comment:
You forgot a space ` :!app.install` in the full colon

Reply:
Have added the space such that it now appears like `: !app.install`